tox.ini

```diff
@@ -24,10 +24,15 @@ commands =
     bandit -r ipaserver/ -ll
 
 [testenv:jslint]
+deps =
+skipsdist=
```

Comment:
Shouldn't this have a value, like true ?

Reply:
It works without an argument. I changed the line to ``skipdist = true`` to make it obvious.